Src/BootCamp.Chapter/CredentialsManager.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    internal class CredentialsManager
+    {
+        private const string CredentialsPath = @"..\..\..\Database\Credentials.txt";
+
+        public CredentialsManager()
+        {
+            if (!File.Exists(CredentialsPath))
+            {
+                File.WriteAllText(CredentialsPath, "");
+            }
+        }
+
+        public void Register()
+        {
+            if (RegisterUser())
+            {
+                Console.WriteLine("You have successfully registered.\n");
+            }
+            else
+            {
+                Console.WriteLine("Sorry, your registration was unsuccessful.  The username is already taken.\n");
+            }
+        }
+
+        private bool RegisterUser()
```

Comment:
Split persistence of credentials and getting input

Reply:
This is probably referring to the input and business logic being bunched together.  That change has been undone.